Deletable.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Deletable.swift
+//  pilot
+//
+//  Created by Nick Eckert on 10/4/17.
+//  Copyright © 2017 sanction. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+import ObjectMapper
+import AlamofireObjectMapper
+
+// The fetchable protocol will allow an object to be capable of fetching data from a URL
```

Comment:
Wrong comment

Reply:
You can see how similar fetchable is to deletable.